### PR TITLE
first test

### PR DESCRIPTION
**Pull request by iulusoy**

### CODE REVIEW

File: our_package/test/data/test_data.csv [hunk 1]
```diff
@@ -0,0 +1,102 @@
+ time   norm   <x>   <y>   <z>   <H>
```
Comment (iulusoy) [inline our_package/test/data/test_data.csv:1]: this is a test file for comparison and should live in test/data - later when publishing the package you can also include data files in the distribution like this

File: our_package/test/test_analysis.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import analysis
+import pytest
+import pandas as pd
+
+
+@pytest.fixture
```
Comment (iulusoy) [inline our_package/test/test_analysis.py:6]: a fixture to set up an initial dataframe

File: our_package/test/test_statistics.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import statistics
+import settings
+import pandas as pd
+
+
+# using the tmp_path pytest fixture for temporary paths
+def test_plot_relevant(tmp_path):
```
Comment (iulusoy) [inline our_package/test/test_statistics.py:7]: this uses the tmp_path fixture from pytest/pathlib to generate temporary paths that are "destroyed" after the tests are run

**Review by iulusoy — COMMENTED** [review]
Please take a look at changes

**Review by pip-chang — APPROVED** [review]

**Review by timomit — APPROVED** [review]

**Pull request merged**